docs/man/nng_msg_capacity.3.adoc

```diff
@@ -0,0 +1,46 @@
+= nng_msg_capacity(3)
```

Comment:
Can you add a few more cross references back to these pages?

Specifically in nng_msg.5.adoc, and nng_msg_len.3.adoc and nng_msg_realloc.3.adoc?

Reply:
There's already a cross-reference in realloc (possibly I missed the list at the bottom, though).  I'll attend to the others.